Directory.Build.props

```diff
@@ -15,6 +15,9 @@
 
     <!-- Allow references to unsigned assemblies (like MAUI) from signed projects -->
     <NoWarn>$(NoWarn);CS8002</NoWarn>
+
+    <!-- Platform-agnostic TFMs that packages should target. -->
+    <SentryMainTargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462</SentryMainTargetFrameworks>
```

Comment:
hm, I find a lot of value in seeing the TFMs on the projects instead of having to refer to this.
We usually don't need to think of aligning across many, it's a per TFM thing

Reply:
My understanding was that when packages are being imported by consumers as a primary dependency, it needs to have the right TFMs. By defining these in a single place, we won't have an issue where a lower TFM from Sentry package would be pulled-in transitively.

Reply:
If we could keep the `MainTFMs` on all of them that'd be great. But because of the mono repo with the legacy stuff mixed in we have to special-case projects individually anyway. Let's be explicit and keep them on the projects.

Reply:
ok, changed to explicit TFMs

Reply:
Thank you